src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -214,6 +216,39 @@ public ITextComponent getShortDisplayString()
         }
     }
 
+    public static class SmeltAbleOreRequest extends AbstractRequest<SmeltableOre>
+    {
+
+        private static final ImmutableList<ItemStack> oreExamples = ImmutableList.of(new ItemStack(Blocks.GOLD_ORE), new ItemStack(Blocks.IRON_ORE));
+
+        SmeltAbleOreRequest(@NotNull final IRequester requester, @NotNull final IToken token, @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, requested);
+        }
+
+        SmeltAbleOreRequest(
+                @NotNull final IRequester requester,
+                @NotNull final IToken token,
+                @NotNull final RequestState state,
+                @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, state, requested);
+        }
+
+        @NotNull
+        @Override
+        public ITextComponent getShortDisplayString()
+        {
+            return new TextComponentTranslation(TranslationConstants.COM_MINECOLONIES_REQUESTS_SMELTABLE_ORE);
+        }
+
+        @Override
+        public List<ItemStack> getDisplayStacks()
+        {
+            return oreExamples;
+        }
+    }
+
     public static class BurnableRequest extends AbstractRequest<Burnable>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 43 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)